Sources/JWTKit/ECDSA/ECDSAKey.swift

```diff
@@ -190,3 +199,197 @@ extension ECDSAKey.Curve {
         }
     }
 }
+
+public protocol ECDSACurveType: Sendable {
+    static var curve: ECDSAKey.Curve { get }
+}
+
+extension P256: ECDSACurveType, @unchecked @retroactive Sendable {
+    static public var curve: ECDSAKey.Curve { .p256 }
+}
+public typealias ES256PublicKey = ECDSA.PublicKey<P256>
+public typealias ES256PrivateKey = ECDSA.PrivateKey<P256>
+
+extension P384: ECDSACurveType, @unchecked @retroactive Sendable {
```

Comment:
These will need versions for Swift 5 and Swift 6

Reply:
I thought it wouldn’t, because this version of the library would always be compiled with the Swift 5 language mode. Once v5 is used, then none of this is included.

Reply:
Ah, nvm, forgot retroactive didn’t exist pre swift 6 compiler, will update

Reply:
Though I'm now wondering if its worth adding Sendable conformances at all, since the rest of the library didn't support it yet _at all_…

Reply:
I don't mind not having them tbh, anyone wanting a full sendable supported library can bump to v5